inc/class-stream-wrapper.php

```diff
@@ -0,0 +1,1210 @@
+<?php
+
+namespace S3_Uploads;
+
+use Aws\CacheInterface;
+use Aws\LruArrayCache;
+use Aws\Result;
+use Aws\S3\Exception\S3Exception;
+use Aws\S3\S3ClientInterface;
+use Exception;
+use GuzzleHttp\Psr7;
+use GuzzleHttp\Psr7\CachingStream;
+use GuzzleHttp\Psr7\Stream;
+use Psr\Http\Message\StreamInterface;
```

Comment:
New phpcs error.

Reply:
I'm using it in PHPDOc, thikning maybe that's a PHPCS bug? I can ignore the error though.

Reply:
Done